### ISSUE TITLE
Move URI rewriting from Minify_CSS

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/minify#54]
```
Олег Черній has contributed some code dealing with URI rewriting and 
I 
think it'd be a good goal to remove the rewriting functionality from 
Minify_CSS since rewriting is not a goal of minification. 

UriRewriter.php will have methods like "fileToRootRelative" and 
"rootToFileRelative" and a new CSS minifier will be composed from 
Minify_CSS and UriRewriter to serve most of Minify::serve's needs, just as 
Minify_Javascript wraps JSMin.
```

---

Original issue reported on code.google.com by `mrclay....@gmail.com` on 3 Oct 2008 at 8:33

**Comment by GoogleCodeExporter** [comment]
```
R286: rewrote Minify_CSS using separate compressor and urirewriter classes. 
Unit 
tests pass.
@todo: more testing
```

---

Original comment by `mrclay....@gmail.com` on 28 Jan 2009 at 10:08
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Released in 2.1.2
```

---

Original comment by `mrclay....@gmail.com` on 2 Mar 2009 at 10:48
- Changed state: **Verified**
- Added labels: **Release-2.1.1**
